internal/services/vpcgw/network.go

```diff
@@ -69,14 +76,15 @@ func ResourceNetwork() *schema.Resource {
 			"enable_dhcp": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Default:     true,
 				Description: "Enable DHCP config on this network",
+				Deprecated:  "Please use ipam_config",
```

Comment:
Add a link to the migration guide with a more explicit guide that IPAM is the way forward.